### DISCUSSION TITLE
Github Stats Bug - Only showing half of my commits

### DISCUSSION THREAD

**Discussion by MusaAqeel** [anuraghazra/github-readme-stats#2221] [Q&A] [answered]
### Describe the bug

Its showing me only half of my actual commits. https://github.com/MusaAqeel/MusaAqeel

### Expected behaviour

It should show all 345 commits, but its only showing around half.

**Comment by rickstaa** [comment, marked as answer]
@MusaAqeel Thanks for your question. I checked your profile, and the card is working correctly. You can check this out by using the following syntax in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
  }
}
```
You can then use the following in the query variables field.

```graphql
{"login": "MusaAqeel"}
```

You can get the private commits to show up using the `count_private=true` variable. Also, please be aware that we use caching to reduce the server load (see https://github.com/anuraghazra/github-readme-stats#common-options).

**Reply by MusaAqeel** [reply]
thanks, much appreciated.

**Reply by rickstaa** [reply]
@MusaAqeel, could you maybe accept the answer if it is answered? This helps me keep track of unanswered answers.

**Reply by MusaAqeel** [reply]
@rickstaa Apologizes.
